readme.md

```diff
@@ -1,235 +1,369 @@
 
 
+<br />
+
+# <div align="center">$\textcolor{deeppink}{\textsf{Plugin Support}}$</div> <a name="plugin-support" />
 ---
-# Plugin Support
 
-EOS Plugin for Unity API Documentation can be found at https://eospluginforunity.playeveryware.com
+Epic Online Services Plugin for Unity API Documentation can be found at https://eospluginforunity.playeveryware.com
 
-For issues related to integration or usage of the Unity plugin, please create a ```New Issue``` under the [Issues](https://github.com/PlayEveryWare/eos_plugin_for_unity/issues) tab.
+For issues related to integration or usage of the Unity plugin, please create a ```New Issue``` under the [Issues](https://github.com/PlayEveryWare/eos_plugin_for_unity/issues) tab in the github repo.
 
-For issues related to Epic Online Services SDK, Epic Dev Portal or general EOS SDK information, please go to [Epic Online Services Community Support](https://eoshelp.epicgames.com/)
+For issues related to Epic Online Services SDK, Epic Dev Portal or general EOS SDK information, please go to [Epic Online Services Community Support](https://eoshelp.epicgames.com/).
 
 Detailed descriptions and usage for EOS SDK Interfaces, can be found at [EOS Developer Documentation: Game Services](https://dev.epicgames.com/docs/services/en-US/GameServices/index.html).
 
+<br />
 
+# <div align="center">$\textcolor{deeppink}{\textsf{Source Code Contributor Notes}}$</div> <a name="source-code-contributor-notes" />
 ---
-# Source Code Contributor Notes
 
-The following are guidlines for helping contribute to this open source project.
+The following are guidelines for helping contribute to this open source project.
+
+## Contributor Prerequisites
 
-## Prerequisites
 * Ensure At least Visual Studio 2017 is installed.
-* Ensure At least Unity 2020.1.11f1 is installed
-* Ensure required Platform SDKs are installed (Windows, Linux, macOS, Android, iOS, Consoles)
+* Ensure At least Unity 2020.1.11f1 is installed.
+* Ensure required Platform SDKs are installed (Windows, Linux, macOS, Android, iOS, Consoles).
 
 ## Build steps For Native Libraries
-* Build the Visual Studio solutions for the native DLLs
-    * Build the <code>DynamicLibraryLoaderHelper</code> sln in DynamicLibraryLoaderHelper/ for all platforms
-    * (Currently needed) Build the UnityEditorSharedDictionary sln in UnityEditorSharedDictionary/ for all platforms
-        * Only the NativeSharedDictionary project is needed.  
 
-A successful build will mean the correct binaries have been placed in the proper locations for Unity to successfully initialize EOS SDK.
+ Build the Visual Studio solutions for the native DLLs, extra platform specific instructions may be located in the docs for that platform.
+
+1. In your local repository, navigate to the ```DynamicLibraryLoaderHelper``` folder of your platform choice in [NativeCode](NativeCode).
+
+   > :heavy_exclamation_mark: These files are not included with the package imported via tarball or git url.
```

Comment:
Perhaps mention to clone the repo for this section

Reply:
added it as a prereq in that section